src/Host/Broker/Impl/Pipes/ProcessHelpers.cs

```diff
@@ -10,6 +10,7 @@
 
 namespace Microsoft.R.Host.Broker.Pipes {
     internal static class ProcessHelpers {
+        /*
```

Comment:
Do we still want to keep this code around, or are we sure we're not going to need this at this point? If the latter, might as well remove it...